auto_py_to_exe/web/js/i18n.js

```diff
@@ -2068,6 +2136,10 @@ const supportedLanguages = [
     name: 'Greek (Ελληνικά)',
     code: 'el',
   },
+  {
```

Comment:
Can you add this new language to the README please

Reply:
Sure. Sorry for the late response.

Reply:
Sorry, I meant a line in the translation table in the README